bindings/matrix-sdk-ffi/src/encryption.rs

```diff
@@ -461,6 +461,14 @@ impl UserIdentity {
     pub(crate) fn master_key(&self) -> Option<String> {
         self.inner.master_key().get_first_key().map(|k| k.to_base64())
     }
+
+    /// Is the user identity considered to be verified.
+    ///
+    /// If the identity belongs to another user, our own user identity needs to
+    /// be verified as well for the identity to be considered to be verified.
+    pub fn is_verified(&self) -> bool {
```

Comment:
I am wondering if this data should be live? Maybe not for now but in the future the app could be more reactive if we had a way to get a live `UserIdentity`.

Reply:
Yeah, I'd really like this to be a property on `RoomMember` that would update automatically through the member's stream if I'm honest, but I don't have a clue how to do that as the members live separately to the encryption 🫤